packages/ethereum/deploy/02_HoprToken.ts

```diff
@@ -16,41 +16,30 @@ const main: DeployFunction = async function ({
 
   const deployer = await getNamedAccounts().then((o) => ethers.getSigner(o.deployer))
 
-  const result = await deployments.deploy('HoprToken', {
+  const tokenContract = await deployments.deploy('HoprToken', {
     from: deployer.address,
     log: true
   })
 
   if (network.tags.testing || network.tags.development) {
-    const hoprToken = (await ethers.getContractFactory('HoprToken')).attach(result.address) as HoprToken
+    const hoprToken = (await ethers.getContractFactory('HoprToken')).attach(tokenContract.address) as HoprToken
     const MINTER_ROLE = await hoprToken.MINTER_ROLE()
     const isDeployerMinter = await hoprToken.hasRole(MINTER_ROLE, deployer.address)
 
     // on "testing" networks, we cannot wait 10 blocks as there is no auto-mine
     // on "development" networks, we must wait 10 blocks since hardhat is not aware of the txs
     if (!isDeployerMinter) {
       console.log('Granting MINTER role to', deployer.address)
-      const grantRoleTx = hoprToken.grantRole(MINTER_ROLE, deployer.address)
-      if (network.tags.development) {
-        await (await grantRoleTx).wait(10)
-      } else {
-        await grantRoleTx
-      }
+      await hoprToken.grantRole(MINTER_ROLE, deployer.address)
 
       if (mintedTokenReceiver) {
         console.log('Minting tokens to', mintedTokenReceiver)
-        const mintTx = hoprToken.mint(
+        await hoprToken.mint(
```

Comment:
What is the reason for not waiting 10 blocks anymore in both cases?

Reply:
It is not necessary - hardhat does the same thing, might have been a compatibility thing in the past

Reply:
Checked, need to remove the comment

Reply:
removed comment